octodns_route53/provider.py

```diff
@@ -1234,6 +1234,9 @@ def populate(self, zone, target=False, lenient=False):
                     else:
                         aliases[record_name].append(rrset)
                     continue
+                if 'TrafficPolicyInstanceId' in rrset:
```

Comment:
It's a minor nit, but I think it'd make sense to make this an elif since it fits in with the other skips above and wouldn't happen if any of those checks were True

```suggestion
                elif 'TrafficPolicyInstanceId' in rrset:
```

Reply:
Totally make sense, applied